src/group/directive.ts

```diff
@@ -11,13 +11,19 @@ interface CustomEvent extends Event { }
 
 @Directive({
   // tslint:disable-next-line:directive-selector
-  selector: 'form[ngrxFormState]',
+  selector: 'form:not([ngrxFormsAction])[ngrxFormState]',
```

Comment:
i'd prefer `form[ngrxFormState]:not([ngrxFormsAction])`

Reply:
Tried that initially, but the selector won't work this way.